src/App.tsx

```diff
@@ -63,49 +63,53 @@ export class App extends React.Component<{}, State> {
       this.state,
     );
 
+    const {
+      isReversed,
+      sortType,
+    } = this.state;
+
     return (
       <div className="section content">
         <div className="buttons">
           <button
             type="button"
-            className={`button is-info${this.state.sortType !== 1 ? ' is-light' : ''}`}
-            onClick={() => this.setState({ sortType: 1 })}
+            className={`button is-info${sortType !== SortType.ALPHABET ? ' is-light' : ''}`}
+            onClick={() => this.setState({ sortType: SortType.ALPHABET })}
           >
             Sort alphabetically
           </button>
 
           <button
             type="button"
-            className={`button is-success${this.state.sortType !== 2 ? ' is-light' : ''}`}
-            onClick={() => this.setState({ sortType: 2 })}
+            className={`button is-success${sortType !== SortType.LENGTH ? ' is-light' : ''}`}
```

Comment:
you could also used the `classnames` package for conditional classes assignment:
https://www.npmjs.com/package/classnames

it's pretty useful - in Mate we use it super often